wordpress-mobile/WordPress-iOS
# Improve site creator tests

Fixes #NA

This PR improves site creator tests and restores private access to the `onContinue` closure in `SiteNameView`

To test:

- checkout and build tests on this branch
- run `SiteCreatorTests` and make sure all tests succeed
- optionally, run the entire unit test suite and make sure no test fails

## Regression Notes
1. Potential unintended areas of impact
na

2. What I did to test those areas of impact (or what existing automated tests I relied on)
na

3. What automated tests I added (or what prevented me from doing so)
na

PR submission checklist:
